spark/src/test/scala/org/apache/spark/sql/delta/DescribeDeltaHistorySuite.scala

```diff
@@ -139,6 +141,21 @@ trait DescribeDeltaHistorySuiteBase
       .asInstanceOf[Map[String, String]]
   }
 
+  // Returns necessary delta property json expected for the test. If coordinated commit is enabled,
+  // a few properties will be automatically populated, and this method will take care of it.
+  protected def getPropertiesJson(extraProperty: Option[Map[String, String]] = None): String = {
+    val coordinatedCommitsProperty = if (coordinatedCommitsEnabledInTests) {
+      getCoordinatedCommitsDefaultProperties()
```

Comment:
Why is this call-site needs `withICT` as false while the other one in `DescribeDeltaDetailSuite.scala` passes it as true?

Reply:
`DESCRIBE DETAIL` returns ICT information as well, but HISTORY doesn't. The helper just evolves to cover both cases.